frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/__tests__/EventListenerDetails.spec.tsx

```diff
@@ -28,9 +28,9 @@ describe('EventListener Details', () => {
     );
   });
 
-  it('should not render EventListenerTriggers section if the trigger contains binding ref', () => {
+  it('should render EventListenerTriggers section if the trigger contains binding ref', () => {
     wrapper.setProps({ obj: EventlistenerTestData[EventlistenerTypes.BINDINGS_TEMPLATE_REF] });
-    expect(wrapper.find(EventListenerTriggers).exists()).toBe(false);
+    expect(wrapper.find(EventListenerTriggers).exists()).toBe(true);
   });
 
```

Comment:
I think we need another test here for `name` to successfully find `EventlistenerTriggers`